lib/providers/services/eos_service.dart

```diff
@@ -94,8 +94,35 @@ class EosService {
     return client.pushTransaction(transaction, broadcast: true);
   }
 
+  Future<dynamic> plantSeeds({String amount}) async {
+    print("[eos] plant seeds ($amount)");
+
+    if (mockEnabled) {
+      return HttpMockResponse.transactionResult;
+    }
+
+    Transaction transaction = buildFreeTransaction([
+      Action()
+        ..account = "token.seeds"
+        ..name = "transfer"
+        ..authorization = [
+          Authorization()
+            ..actor = accountName
+            ..permission = "active"
+        ]
+        ..data = {
+          "from": accountName,
+          "to": "harvst.seeds",
+          "quantity": "$amount SEEDS",
+          "memo": "",
+        }
+    ]);
+
+    return client.pushTransaction(transaction, broadcast: true);
+  }
+
   Future<dynamic> createInvite(
-      {double transferQuantity, double sowQuantity, String inviteHash}) async {
+      {double transferQuantity, double sowQuantity, double inviteHash}) async {
```

Comment:
ummm how is inviteHash a double value???

Reply:
fixed